### PR TITLE
Added editable alerting-related fields to the user profile

### LINKED ISSUE #92: Synchronize user media in Zabbix according to user profile settings in esdc

### PR DESCRIPTION
**Pull request by secult**
Also media_severity_generator is fixed.

Closes #92

### CODE REVIEW

**Comment by secult** [comment]
Let me know in case I should add something to changelog, or pls put it there by yourself if it's in a hurry.

File: gui/migrations/0004_alerting_fields_initialization.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+
+
+# noinspection PyUnusedLocal
+def initialize_alerting_fields(apps, schema_editor):
+    former_user_profile = apps.get_model('gui', 'UserProfile')
+    if former_user_profile.objects.count() > 10000:
+        warning_text = "\n It looks like there is a lot of users in your database and " \
```
Comment (dn0) [inline gui/migrations/0004_alerting_fields_initialization.py]: Single quotes please

File: gui/migrations/0004_alerting_fields_initialization.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                       "it would take a lot of time to update their profiles. This migration is therefore skipped. " \
+                       "If you need to, perform this operation manually."
```
Comment (dn0) [inline gui/migrations/0004_alerting_fields_initialization.py]: -> `Please perform this operaion manually.`

File: gui/migrations/0004_alerting_fields_initialization.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        Warning(warning_text)
```
Comment (dn0) [inline gui/migrations/0004_alerting_fields_initialization.py]: Line has no effect -> remove

File: gui/templates/gui/dc/profile/profile_page.html [hunk 4]
```diff
@@ -65,6 +65,14 @@
       <div id="profile_sshkey_list">
         {% include "gui/profile/profile_sshkey_list.html" %}
       </div>
+
+      <div class="tab-header">
+        <i class="icon-envelope"></i> {% trans "Alerting methods" %}
```
Comment (dn0) [inline gui/templates/gui/dc/profile/profile_page.html]: Lets call this Alerting Methods or Alerting Media
(capital `M`)

File: gui/templates/gui/profile/profile_page.html [hunk 5]
```diff
@@ -85,6 +85,13 @@
       <div id="profile_sshkey_list">
         {% include "gui/profile/profile_sshkey_list.html" %}
       </div>
+
```
Comment (dn0) [inline gui/templates/gui/profile/profile_page.html]: Move this profile form block under the `billing_address_details` block.

**Pull request merged**